lib/api/lambda.js

```diff
@@ -100,6 +100,17 @@ Lambda.prototype.scheduledEvent = function(path, handler) {
 };
 
 /**
+ * Routes API GATEWAY AUTHORIZER EVENT requests for a given rule to the handler. 
```

Comment:
Does the user need to configure API gateway manually to get it to send these events?